bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/EntitySubGridSteps.cs

```diff
@@ -111,7 +111,11 @@ public static void ThenICanNotSeeInTheSubgrid(string alias, string subGridName)
         [Then(@"I can see (exactly|more than|less than) (\d+) records in the '(.*)' subgrid")]
         public static void ThenICanSeeRecordsInTheSubgrid(string compare, int count, string subGridName)
         {
-            var actualCount = XrmApp.Entity.SubGrid.GetSubGridItemsCount(subGridName);
+            //var actualCount = XrmApp.Entity.SubGrid.GetSubGridItemsCount(subGridName);
+            //Temporary until XrmApp.Entity.SubGrid.GetSubGridItemsCount(subGridName) works for a count of 0
```

Comment:
Please could we add a link to the EasyRepro issue.
```suggestion
            // Temporary until XrmApp.Entity.SubGrid.GetSubGridItemsCount(subGridName) works for a count of 0
            // https://github.com/microsoft/EasyRepro/issues/1318
```